sdks/python/apache_beam/coders/coder_impl.py

```diff
@@ -435,6 +435,23 @@ def estimate_size(self, unused_value, nested=False):
     return 8
 
 
+class TimerCoderImpl(StreamCoderImpl):
+  """For internal use only; no backwards-compatibility guarantees."""
+  def __init__(self, payload_coder_impl):
+    self._timestamp_coder_impl = TimestampCoderImpl()
+    self._payload_coder_impl = payload_coder_impl
+
+  def encode_to_stream(self, value, out, nested):
+    self._timestamp_coder_impl.encode_to_stream(value['timestamp'], out, True)
+    self._payload_coder_impl.encode_to_stream(value.get('payload'), out, True)
+
+  def decode_from_stream(self, in_stream, nested):
+    return dict(
```

Comment:
Do we eventually want a concrete class type here?  If so, can you add a TODO or comment either way?

Reply:
Done.